subprojects/workers/src/main/java/org/gradle/workers/internal/WorkerDaemonExpiration.java

```diff
@@ -29,6 +29,7 @@
 public class WorkerDaemonExpiration implements MemoryHolder {
 
     private static final Logger LOGGER = Logging.getLogger(WorkerDaemonExpiration.class);
+    public static final String DISABLE_EXPIRATION_PROPERTY_KEY = "org.gradle.workers.disable-daemons-expiration";
```

Comment:
Let's put 'internal' in the property name. This is very much not something we want to make public.